frontend/src/components/pod/List.tsx

```diff
@@ -77,6 +77,42 @@ export function PodListRenderer(props: PodListProps) {
           return podRow.reason;
         },
       },
+      {
+        label: t('frequent|Pod IP'),
+        getter: (pod: Pod) => pod.status.podIP,
+        sort: (p1: Pod, p2: Pod) => {
+          if (p1.status.podIP < p2.status.podIP) {
+            return -1;
+          } else if (p1.status.podIP > p2.status.podIP) {
+            return 1;
+          }
+          return 0;
+        },
+      },
+      {
+        label: t('frequent|Node Name'),
+        getter: (pod: Pod) => pod.spec.nodeName,
+        sort: (p1: Pod, p2: Pod) => {
+          if (p1.spec.nodeName < p2.spec.nodeName) {
+            return -1;
+          } else if (p1.spec.nodeName > p2.spec.nodeName) {
+            return 1;
+          }
+          return 0;
+        },
+      },
+      {
+        label: t('frequent|Node IP'),
```

Comment:
This is not showing in the regular `kubectl get pods -o wide`, so I think we should enable this from a plugin instead of by default.